astropy/wcs/docstrings.py

```diff
@@ -2232,6 +2232,29 @@ def RA_DEC_ORDER(indent=0):
 Sect. 5.2.1), integer (Sect. 5.2.3), and floating-point values
 (Sect. 5.2.4) for all keywords.
 
+
+.. warning::
+    There are two possible ways of writing to the attributes on this class::
+
+        Wcsprm.crval = [crval]
+        Wcsprm.cdelt = [cdelt]
+        Wcsprm.ctype = [ctype]
+        Wcsprm.cunit = [cunit]
+
+    or::
+
+        Wcsprm.crval[0] = crval
```

Comment:
and modify this example as 
```suggestion
        Wcsprm.crval[0] = crval1
```